netbox/netbox/configuration_example.py

```diff
@@ -190,6 +190,9 @@
 #     }
 # }
 
+# Expose the SWAGGER UI schemas to unauthenticated users
+PUBLIC_SWAGGER = True
+
```

Comment:
We can omit this from the example configuration. It's not meant to be an exhaustive list.

Reply:
ack, done in the new commit.